Formula/fish.rb

```diff
@@ -22,9 +22,16 @@ class Fish < Formula
 
   def install
     system "autoconf" if build.head? || build.devel?
+    args = %W[
+      --prefix=#{prefix}
+      --with-extra-functionsdir=#{HOMEBREW_PREFIX}/share/fish/vendor_functions.d
```

Comment:
Fixed whitespace.

now formula is failing audit
```
ish:
  * Use pkgshare instead of (share/"fish")
  * Use pkgshare instead of (share/"fish")
  * Use pkgshare instead of (share/"fish")
Error: 3 problems in 1 formula
```

How can I bypass this check? We have to create directly.